plugins/cost-insights/src/hooks/useCurrency.tsx

```diff
@@ -34,8 +33,11 @@ export const CurrencyContext = React.createContext<
 >(undefined);
 
 export const CurrencyProvider = ({ children }: PropsWithChildren<{}>) => {
-  const engineers = findAlways(defaultCurrencies, c => c.kind === null);
-  const [currency, setCurrency] = useState<Currency>(engineers);
+  const config = useConfig();
```

Comment:
Good find here, a bit strange that `CurrencyProvider` wasn't fetching from config before 😱